apps/settings/js/panels/root/stk_item.js

```diff
@@ -0,0 +1,161 @@
+define(function(require) {
+  'use strict';
+  var STKHelper = require('shared/stk_helper');
+
+  function STKItem() {
+    this.init();
+  }
+
+  STKItem.prototype.init = function() {
+    var iccMainHeader = document.getElementById('icc-mainheader');
+    var iccEntries = document.getElementById('icc-entries');
```

Comment:
then we can use `this._elements.iccMainHeader` and `this._elements.iccEntries` to do anything we want later.